packages/react-notion-x/src/components/header.tsx

```diff
@@ -13,12 +13,14 @@ import { SearchDialog } from './search-dialog'
 
 export const Header: React.FC<{
   block: types.CollectionViewPageBlock | types.PageBlock
-}> = ({ block }) => {
+  children?: React.ReactNode
+}> = ({ block,children }) => {
   return (
     <header className='notion-header'>
       <div className='notion-nav-header'>
         <Breadcrumbs block={block} />
         <Search block={block} />
+        {children}
```

Comment:
This will not work as, the container className have `justify-content: space-between` which will push the Search button in the middle.

If you wanted to give placeholder for avatar this is not the correct solution.